packages/contracts/contracts/interfaces/ISemaphoreZk3.sol

```diff
@@ -0,0 +1,104 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.4;
+
+/// @title SemaphoreVoting interface.
+/// @dev Interface of SemaphoreVoting contract.
+interface ISemaphoreZk3 {
+    error Semaphore__CallerIsNotCoordinator();
+    error Semaphore__MerkleTreeDepthIsNotSupported();
+    error Semaphore__YouAreUsingTheSameNillifierTwice();
+
+    struct Verifier {
+        address contractAddress;
+        uint256 merkleTreeDepth;
+    }
+
+    struct Circle {
+        address coordinator;
+        string contentURI;
+        bool doubleSpend;
+    }
+
+    /// @dev Emitted when a new group is created.
+    /// @param circleId: Id of the group.
+    /// @param coordinator: Coordinator of the group.
+    event CircleCreated(uint256 circleId, address indexed coordinator);
+
+    /// @dev Emitted when a user is added to a group.
+    /// @param circleId: Id of the group.
+    /// @param identityCommitment: identity added.
+    event IdentityAdded(uint256 indexed circleId, uint256 identityCommitment);
+
```

Comment:
```suggestion
```

This event is not used. `SemaphoreGroups.sol` already emits `MemberAdded` with the same parameters plus `index` and `merkleTreeRoot`.